start-up/permission-tutorial.md

```diff
@@ -1,3 +1,32 @@
 # Permission Tutorial
 
-TODO
\ No newline at end of file
+Discord has a built-in permission system to configure who can or cannot execute certain commands of the bot. This tutorial will guide you through the process of setting up permissions for MoonlightBot.
+
+## Open the server settings
+
+1. To start setting up permissions, you need to open the server settings. Click on the server name to open the server menu, then click on "Server Settings".
+2. In the server settings, click on "Integrations" in the left sidebar.
+3. Here you'll see a list of all the bots and integrations in your server. Click on the "MoonlightBot" entry to open the bot command permissions menu.
+![PermissionTutorialStep1](<../.gitbook/assets/PermissionTutorialStep1.png>)
+
+## Set up permissions
+
+{% hint style="info" %}
+
+### Remember to save your changes!
```

Comment:
Headers are not to make text bigger, they mark sections of the page. A simple bold will work.